recipes/screen/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "screen" %}
+{% set version = "4.8.0" %}
+{% set version_printed = "4.08.00" %}  # what screen -v prints
+{% set sha256 = "6e11b13d8489925fde25dfb0935bf6ed71f9eb47eff233a181e078fde5655aa1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "./configure --prefix=${PREFIX} && make && make install"
```

Comment:
This usually goes in a build.sh script.  Also, please use `make -j ${CPU_COUNT}`